### PR TITLE
[WIP] Schedule transactional role specific tests

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
- Related ticket: https://progress.opensuse.org/issues/42929

## Verification run:

| Distribution     | transactional_host_install_only | transactional_host |
| ----------------- | --------------------------------------- | ----------------------- |
| Tumbleweed  | [slindomansilla-vm#1111](http://slindomansilla-vm.qa.suse.de/tests/1111) | (coming soon) |
| Leap 15.1 | (coming soon)  | (coming soon) |
| SLE 15.1 | (coming soon)  | (coming soon) |

- kubeadm: [slindomansilla-vm#1105](http://slindomansilla-vm.qa.suse.de/tests/1105)
- Kubic/MicroOS: (coming soon)

### CODE REVIEW

**Comment by SergioAtSUSE** [comment]
Fix for SCHEDULE syntax. No need to write tests/ prefix: https://github.com/os-autoinst/os-autoinst/pull/1071

File: tests/caasp/filesystem_ro.pm [hunk 1]
```diff
@@ -14,16 +14,17 @@
...
 sub run {
+    select_console 'root-console' if (check_var('DISTRI', 'opensuse'));
     assert_script_run "! touch /should_fail";
     assert_script_run "touch /etc/should_succeed";
     assert_script_run "touch /var/log/should_succeed";
 
     assert_script_run 'btrfs property get / ro | grep "ro=true"';
 
-    if (is_caasp '4.0+') {
+    if (is_caasp('4.0+') || is_leap('15.0+') || is_tumbleweed) {
```
Comment (okurz) [inline tests/caasp/filesystem_ro.pm]: Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.

File: tests/caasp/filesystem_ro.pm [hunk 2]
```diff
@@ -14,16 +14,17 @@
 use base "opensusebasetest";
```
Comment (okurz) [inline tests/caasp/filesystem_ro.pm]: I guess if you can reuse the module outside the context of "caasp" please also move it out of the "caasp" subfolder. Maybe we can have a new folder "transactional" which is used by the corresponding roles, caasp, microos and kubic?

**Comment by SergioAtSUSE** [comment]
Related: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6626

**Comment by SergioAtSUSE** [comment]
Already fixed by mkravec in https://progress.opensuse.org/issues/51296#note-17

**Pull request closed without merging**